landscape/converter/mongo_to_bigartm_converter.py

```diff
@@ -0,0 +1,169 @@
+"""
+Convert Github daily issues to one of BigARTM format
+(UCI Bag-of-words https://bigartm.readthedocs.io/en/stable/tutorials/datasets.html)
+"""
+import argparse
+from datetime import datetime
+import os
+from typing import List, Optional
+
+import pymongo
+from scipy.sparse import csr_matrix
+from sklearn.feature_extraction.text import CountVectorizer
+import spacy
+from spacy.lang.en.stop_words import STOP_WORDS
+from tqdm import tqdm
+
+from landscape.utils import is_between_dates
+
+stopwords = list(STOP_WORDS)
+english_lang_model = spacy.load("en_core_web_md", disable=["tagger", "parser", "ner"])
+client = pymongo.MongoClient("localhost", 27017)
+issues_landscape_db = client["issues_landscape"]
+
+
+class TokenizedIssue:
+    """
+    The TokenizedIssue object contains issue id and title
+    :param issue_id: Issue id
```

Comment:
let's move it inside `__init__`

Reply:
btw - you can simplify it with [namedtuple](https://docs.python.org/3/library/collections.html#collections.namedtuple) because it just stores something